askbot/templates/question_edit.html

```diff
@@ -11,7 +11,7 @@
     {% if revision_form.revision.errors %}{{ revision_form.revision.errors.as_ul() }}{% endif %}
     {{ revision_form.revision }}
     <input type="hidden" id="select_revision" name="select_revision" value="false"/>
-    <div class="form-item">
+    <div class="form-group">
         <label for="id_title" >
             {{ settings.WORDS_QUESTION_IN_ONE_SENTENCE|escape }}
```

Comment:
We should use the same title "What is your question?" instead of the confusing settings.